internal/services/environment/api_environment.go

```diff
@@ -446,10 +446,15 @@ func (client *Client) doWaitForAccess(ctx context.Context, env *Dto) error {
 		whoAmIUrl := &url.URL{
 			Scheme: constants.HTTPS,
 			Host:   envUrl.Host,
-			Path:   "/api/data/v9.2/WhoAmI",
+			Path:   "/api/data/v9.2/systemusers",
```

Comment:
how systemuser query is better than whoami?

Reply:
Not sure if it is, but thought maybe whoami doesn't need to hit the db and maybe access/role assignments gets checked when you actually read data

Reply:
Ultimately removed this code because standard retry for 401 fixed this error.